# Service to get Charuco Pose

---
float32 x
float32 y
float32 z
float32 q0
float32 qx
float32 qy
float32 qz
